test/test_log.py

```diff
@@ -54,18 +56,13 @@ def test_should_create_correct_logging_link(self):
 
 
     def _assert_single_log_event(self, message_re, severity=None):
-        self.assertEqual(1, self.logging_client.write_log_entries.call_count)
-        calls = self.logging_client.write_log_entries.call_args_list[0][0]
-        le = calls[0][0]
-
-        self.assertEqual(le.resource.type, "global")
```

Comment:
Why is it removed?

Reply:
Resource/labels/etc is not attached to log-events.
It is handled underhood by the 'google.cloud.logging' library.